QUICK_START.md

```diff
@@ -23,6 +23,7 @@ This document provides quick access to several different ways to set up CORTX fo
 5. Build entire CORTX into OVA (virtual machine image): TODO
     1. Run it on a single node: TODO 
     1. Run it on a cluster: TODO
+1. Install CORTX on Supermicro Servers with Seagate RBOD Storage: [LINK](doc/Install_CORTX_on_Supermicro_Servers_with_Seagate_RBOD_Storage.md)
```

Comment:
Until there are at least 2 names in the 'tested by' section we have 'TODO' in this quick start guide. So please change 'LINK' to 'TODO' for now until someone else adds their name to the tested by section. I will try to get this tested once I get the iso files downloaded.

Reply:
I changed it.